cellxgene_schema_cli/requirements.txt

```diff
@@ -1,12 +1,11 @@
-anndata==0.8.0
-cellxgene-ontology-guide==1.0.0
-click==8.1.7
-Cython==3.0.10
-numpy==1.26.4
-pandas==1.4.4
-PyYaml==6.0.1
-wheel==0.43.0
-scipy<1.13.0    # scipy 1.13.0 is not compatible with anndata <=0.10.6, revisit before next release
-semver==3.0.2
-xxhash==3.4.1
-matplotlib==3.9.0
+anndata<0.12
+cellxgene-ontology-guide==1.0.0 # update before a schema migration
+click<9
+Cython<4
+numpy<2
+pandas<3
```

Comment:
I verified this works with 1.4.4 and 2.2.2. The portal still uses 1.4.4, so we will let portal decide what version of pandas is used as long as it's <3